domain/src/main/java/edu/nextstep/camp/calculator/domain/Expression.kt

```diff
@@ -0,0 +1,52 @@
+package edu.nextstep.camp.calculator.domain
+
+import edu.nextstep.camp.calculator.domain.util.CalculatorInputUtil
+
+object Expression {
+    var express: String = ""
```

Comment:
```suggestion
    var express: String = ""
        private set
```
외부에서 해당 값을 설정하지 못하게 할 수 있어요.